src/runtime/linux.rs

```diff
@@ -752,6 +752,9 @@ pub enum LinuxNamespaceType {
 
     /// Network Namespace for isolating network devices, ports, stacks etc.
     Network = 0x40000000,
+
+    /// Time Namespace for isolating the clocks
+    Time = 0x80000000,
```

Comment:
This should be `0x00000080` according to:
https://github.com/torvalds/linux/blob/0983f6bf2bfc/tools/testing/selftests/pidfd/pidfd.h#L26